packages/grafana-ui/src/components/InfoBox/FeatureInfoBox.tsx

```diff
@@ -60,11 +60,26 @@ function getPanelStateBadgeDisplayModel(featureState: FeatureState): BadgeProps
         icon: 'exclamation-triangle',
         color: 'orange',
       };
-  }
 
-  return {
-    text: 'Beta',
-    icon: 'rocket',
-    color: 'blue',
-  };
```

Comment:
The assumption here is that we don't need a default case since the featureState isn't optional?

Reply:
Yeah. Also I am not sure why the code was structured that way - because as you've said, the `featureState` is required and it could have been only `beta` or `alpha`. So in my case, I am just adding a `switch case` for every available option.